src/main/resources/node.linux.json

```diff
@@ -4,27 +4,18 @@
             {
                 "browserName": "firefox",
                 "maxInstances": <MAX_SESSION_FIREFOX>,
-                "instanceId": "<INSTANCE_ID>",
                 "seleniumProtocol": "WebDriver"
             },
             {
                 "browserName": "chrome",
                 "maxInstances": <MAX_SESSION_CHROME>,
-                "instanceId": "<INSTANCE_ID>",
                 "seleniumProtocol": "WebDriver"
             }
         ],
-    "configuration":
-    {
         "proxy": "org.openqa.grid.selenium.proxy.DefaultRemoteProxy",
         "maxSession": <MAX_SESSION>,
-        "uuid": "<UUID>",
-        "instanceId": "<INSTANCE_ID>",
-        "createdDate": "<CREATED_DATE>",
-        "createdBrowser": "<CREATED_BROWSER>",
-        "createdOs": "<CREATED_OS>",
         "register": true,
         "registerCycle": 5000,
-        "hubHost": "<HOST_NAME>"
-    }
+        "hub": "<HOST_NAME>",
+        "custom": {uuid=<UUID>,instanceId=<INSTANCE_ID>,createdDate=<CREATED_DATE>,createdBrowser=<CREATED_BROWSER>,createdOs=<CREATED_OS>}
```

Comment:
For the 'custom' field, can you have each field formatted to be on a new line?  Easier to read that way.